docs/install_old.md

```diff
@@ -6,7 +6,7 @@
 # Installation
 Create a new conda environment for hercules:
 ```
-conda create --name hercules python
+conda create --name hercules python=3.11
```

Comment:
I'm not sure if the python version should be specified here or if there is a way to say here that any version 3.6-3.11 is okay. I know it's specified in `setup.py`, but to my knowledge, that file isn't used at this point in the installation.

Reply:
This is a good point. I think this is OK; in many circumstances, the user will be adding to a larger environment and they'll skip this step, but for someone creating a standalone environment this works